sources/ClangSharp.PInvokeGenerator/PInvokeGenerator.VisitDecl.cs

```diff
@@ -294,6 +295,11 @@ private void VisitEnumConstantDecl(EnumConstantDecl enumConstantDecl)
             parentName = _outputBuilder.Name;
         }
 
+        if (Config.StripEnumMemberTypeName)
+        {
+            escapedName = PrefixAndStrip(escapedName, parentName, trimChar: '_');
```

Comment:
Sorry for the delay, was busy at Microsoft Build.

The rest of the changes look good, but `trimChar: '_'` is going to cause some problems and needs an extra handler. In particular we know that there exist enums of the form `DXGI_FORMAT_420_OPAQUE` and so removing the prefix `DXGI_FORMAT` + `_` will cause an invalid member name to be generated.

I'm going to merge this as is given that these are rare and there is a trivial workaround where devs can simply remap that specific member. However, if you have time it'd be great to at least add in a diagnostic when such a member is encountered and possibly add back the leading `_` to ensure we still get "valid" codegen.

Reply:
Sure thing, Looks like we first need a check for leading digits (i think that's the only problem from the subset of what's valid in C/C++ and C# at that point) and then another call to EscapeName in case we ended up with a keyword after stripping.
I'll make the changes.

In case of leading digits it's fine to just prepend with @ like EscapeName does?